CMakeLists.txt

```diff
@@ -531,26 +531,25 @@ function(unpack_db db_bzip2_file suffix_var)
 
     if(NOT MIOPEN_USE_SQLITE_PERFDB AND __extension STREQUAL ".db")
         add_custom_command(OUTPUT ${KERNELS_BINARY_DIR}/${__fname}.txt
-                           DEPENDS $<TARGET_FILE:sqlite2txt> ${KERNELS_BINARY_DIR}/${__fname}
+                           DEPENDS sqlite2txt generate_${__tname}
                            COMMAND $<TARGET_FILE:sqlite2txt> ${KERNELS_BINARY_DIR}/${__fname} ${KERNELS_BINARY_DIR}/${__fname}.txt
         )
         add_custom_target(generate_${__tname}_txt ALL DEPENDS ${KERNELS_BINARY_DIR}/${__fname}.txt)
         add_dependencies(generate_kernels generate_${__tname}_txt)
-        set("${suffix_var}" ".txt" PARENT_SCOPE)
+        set(__fname ${__fname}.txt)
     else()
         add_dependencies(generate_kernels generate_${__tname})
-        set("${suffix_var}" "" PARENT_SCOPE)
     endif()
+    set(__fname ${__fname} PARENT_SCOPE)
```

Comment:
@apwojcik [Recommendation] I would recommend renaming `__fname` to `unpack_db_output_fname` or something alike to emphasize that value of this variable is produced by unpack_db.

Reply:
@atamazov That makes sense. This PR is already merged. I will prepare a separate PR to change the return value's name.